slurm/datadog_checks/slurm/check.py

```diff
@@ -315,6 +325,30 @@ def process_sdiag(self, output):
 
         self.gauge('sdiag.enabled', 1)
 
+    def process_scontrol(self, output):
+        # PID      JOBID    STEPID   LOCALID GLOBALID
+        # 3771     14       batch    0       0
+        # 3772     14       batch    -       -
+
+        if len(self.scontrol_cmd) > 1:
+            base_cmd = self.scontrol_cmd[:-1]
+        else:
+            base_cmd = self.scontrol_cmd
+        hostname = os.uname()[1]
+        slurm_node = get_subprocess_output(base_cmd + ["show", "hostname", hostname]).strip().decode("utf-8")
```

Comment:
would this command ever fail or return no output?

Reply:
I don't believe this one can.